oauth2_provider/views/oidc.py

```diff
@@ -0,0 +1,86 @@
+import json
+
+from django.http import HttpResponse, JsonResponse
+from django.urls import reverse, reverse_lazy
+from django.utils.decorators import method_decorator
+from django.views.decorators.csrf import csrf_exempt
+from django.views.generic import View
+from jwcrypto import jwk
+
+from ..settings import oauth2_settings
+from .mixins import OAuthLibMixin, OIDCOnlyMixin
+
+
+class ConnectDiscoveryInfoView(OIDCOnlyMixin, View):
+    """
+    View used to show oidc provider configuration information
+    """
+
+    def get(self, request, *args, **kwargs):
+        issuer_url = oauth2_settings.OIDC_ISS_ENDPOINT
+
+        if not issuer_url:
+            abs_url = request.build_absolute_uri(reverse("oauth2_provider:oidc-connect-discovery-info"))
+            issuer_url = abs_url[: -len("/.well-known/openid-configuration/")]
```

Comment:
This is duplicated code from `OAuth2Validator.get_oidc_issuer_endpoint`. Refactor out to a util function.